app/views/application/_mobile-search.html.erb

```diff
@@ -4,8 +4,9 @@
     <!-- <div class="padding"></div> -->
     <div class="navbar-nav">
       <form action="https://librarysearch.temple.edu/bento" class="mobile">
+        <label for="q" class="visuallyhidden">Search Everything</label>
         <input type="text" name="q" id="librarySearch" class="mobile" placeholder="Search everything" autocomplete="off" />
-        <input type="submit" class="mobile" value = "">
+        <input type="submit" class="mobile" value = "" aria-label="submit search query">
```

Comment:
@cdoyle-temple Why is the first input using a `visuallyhidden` label tag, but the second  uses an inline `aria-label` attribute? Not criticism, but asking what the difference is?

Reply:
The first is a label for the search field itself. We can hide that as it isn't part of the design. The second is a label for the submit button which we can't really hide.